frontend/src/components/MyCredentialList.vue

```diff
@@ -106,10 +95,21 @@ export default {
               this.data = result.data.filter((item) => {
                 return item.issuer;
               });
+
+              if (this.indicateNew) {
+                const newPartners = this.$store.getters.newPartners;
+                if (this.$store.getters.newPartnersCount > 0) {
+                  this.data = this.data.map((partner) => {
+                    if ({}.hasOwnProperty.call(newPartners, partner.id)) {
```

Comment:
Guess this is for testing :)
Should be `newCredentials` here

Reply:
yes, testing ;)